docs/interpreter/cassandra.md

```diff
@@ -793,7 +812,17 @@ Below are the configuration parameters and their default value.
 
 <hr/>
 
-## 14. Bugs & Contacts
+## 14. Change Log
+
+**2.0** : 
```

Comment:
I'm pretty attached to the versioning of the **Cassandra** interpreter because it allows users to know what has been changed/added/removed.

 But your remark also makes sense because this specific versioning can be confusing for people.

 The best strategy would be to specify the interpreter version along side with the Zeppelin version (SNAPSHOT or RELEASE does not matter)